packages/wpcom.js/src/index.js

```diff
@@ -198,32 +186,29 @@ WPCOM.Users = Users;
 
 if ( ! Promise.prototype.timeout ) {
 	/**
-	* Returns a new promise with a deadline
-	*
-	* After the timeout interval, the promise will
-	* reject. If the actual promise settles before
-	* the deadline, the timer is cancelled.
-	*
-	* @param {number} delay how many ms to wait
-	* @return {Promise} promise
-	*/
+	 * Returns a new promise with a deadline
+	 *
+	 * After the timeout interval, the promise will
+	 * reject. If the actual promise settles before
+	 * the deadline, the timer is cancelled.
+	 *
+	 * @param {number} delay how many ms to wait
+	 * @returns {Promise} promise
+	 */
 	Promise.prototype.timeout = function( delay = DEFAULT_ASYNC_TIMEOUT ) {
```

Comment:
I think we should remove this `Promise` monkey patch ASAP 🙂

Reply:
Yikes! I didn't even notice what the code below was doing 😬